src/components/Book.js

```diff
@@ -2,11 +2,14 @@ import { CircularProgressbar } from 'react-circular-progressbar';
 import 'react-circular-progressbar/dist/styles.css';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksArrSlice';
 
-const Book = ({ book, handleRemoveBookClick }) => {
-  const handleRemove = () => {
-    handleRemoveBookClick(book.id);
-  };
+const Book = ({ book }) => {
+  const dispatch = useDispatch();
+  // const handleRemove = () => {
+  //   handleRemoveBookClick(book.id);
+  // };
```

Comment:
- [ ] Kindly remove commented code. It can make your code harder to read. You can use git history to view old code.